dtslint/ts3.5/index.ts

```diff
@@ -108,3 +108,9 @@ const gt5 = (n: number) => n > 5;
 R.allPass([odd, lt20, gt5], 20); // $ExpectError
 FR.allPass([odd, lt20, gt5], 20); // $ExpectType boolean
 FR.allPass([odd, lt20, gt5]); // $ExpectType Predicate<number>
+
+// anyPass
+R.anyPass([odd, lt20, gt5], 20); // $ExpectError
+FR.anyPass([odd, lt20, gt5], 20); // $ExpectType boolean
+FR.anyPass([odd, lt20, gt5], 0); // $ExpectType boolean
```

Comment:
as per what I wrote in https://github.com/giogonzo/fp-ts-ramda/pull/24#issuecomment-541328446, this test is not really useful. Let's replace this with an `example` for for the `0` case among jest tests below instead, like I did for `allPass`